### PR TITLE
command to kill/signal processes

### PR DESCRIPTION
**Pull request by brandonbloom**
implements #188

### CODE REVIEW

**Comment by brandonbloom** [comment]
failure appears to be a flakey test, will fix independently

File: cmd/exo/kill.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+var killCmd = &cobra.Command{
+	Use:   "kill [ref...]",
+	Short: "Force stop processes",
+	Long: `Sends a unix signal to process-like components.
+	
+If no refs are provided, signals all processes in the workspace.
+	
+The default signal is SIGKILL.`,
```
Comment (BenElgar) [inline cmd/exo/kill.go:24]: SIGKILL being the default feels strange when other kill commands generally use SIGTERM as the default.

**Review by BenElgar — COMMENTED** [review]
LGTM aside from small comment

File: cmd/exo/kill.go (hunk 1 above)
Comment (brandonbloom) [reply cmd/exo/kill.go:24]: Understood, but 1) `stop` already does sigterm, then timeout, then sigkill. 2) 90% of the time when I use unix `kill`, I use `kill -9` and 3) this is the default in docker compose.

Comment (BenElgar) [reply cmd/exo/kill.go:24]: :+1: Persuaded

**Pull request merged**